modules/relax/models/mixins/hashMap.cfm

```diff
@@ -0,0 +1,25 @@
+<cfscript>
```

Comment:
Jon, we use linked hash maps in coldbox all the time without determining engine, not sure why you are going this route? What's the issue?